core/address/codec.go

```diff
@@ -2,6 +2,8 @@ package address
 
 // Codec defines an interface to convert addresses from and to string/bytes.
 type Codec interface {
+	// GetBech32Prefix returns the bech32 prefix
+	GetBech32Prefix() string
```

Comment:
can we avoid adding bech32 specific to this interface, this interface is meant to be agnostic

Reply:
I see, let me thing of how to do that differently for autocli then 🤔

Reply:
If I rename it to GetAddressPrefix and that it returns "" when there is none, how does it sound for you?